### PR TITLE
Be amazon s3 upload

### PR DESCRIPTION
**Pull request by xia0m**
### What this PR does (required):
- upload photos to Amazon s3
- get Amason s3 image url

### Screenshots / Videos (required):
Screen Shot 2021-05-19 at 1 51 56 AM
Screen Shot 2021-05-19 at 1 52 52 AM

### Any information needed to test this feature (required):
- use postman to post a request to the URL  `localhost:3001/api/upload`
- set the body of the request to form-data
- use 'photos' for the key, and set the types to file
- select single or multiple files and send the request

### Any issues with the current functionality (optional):
- the current max photos is set to 5, when more than 5 photos are selected, there will be an error
- I tried add callback functions in multer, create my own middleware before and after multer, still couldn't figure it out.
Screen Shot 2021-05-19 at 2 00 46 AM

### CODE REVIEW

File: server/routes/upload.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+const multer = require('multer');
+const {uploadPhotos} = require('../controllers/upload');
+// const {check_max_photos} = require('../middleware/upload');
+
+// const {check_max_photos} = require('../middleware/upload')
```
Comment (dproc96) [inline server/routes/upload.js]: You can remove commented out code here

**Review by dproc96 — COMMENTED** [review]
One note on the code, but can you share the env variables in Slack (or point me to them if you already have, I couldn't find them) so that we can test it?

File: server/routes/upload.js (hunk 1 above)
Comment (xia0m) [reply server/routes/upload.js]: Sorry, forgot to share it on Slack, I'll do that right now

**Review by dproc96 — APPROVED** [review]
LGTM

**Comment by dproc96** [comment]
One more note, make sure to run prettier on these files before merging, looks like some of your spacing is off

**Pull request merged**